src/extension.ts

```diff
@@ -512,7 +512,7 @@ async function fileExists(uri: vscode.Uri) {
 	}
 }
 
-async function getSopsGeneralOptions() {
+async function getSopsGeneralOptions(uri: vscode.Uri) {
```

Comment:
I'd give it a little more descriptive name for the variable. Like decryptedFileUri or so. uri is just too general and user of the function can be confused. Or put a JSDoc here.

Reply:
Rename input variable uri => fileUriToEncryptOrDecrypt in commit [1c64602](https://github.com/signageos/vscode-sops/pull/73/commits/1c64602fc4a21a9c9863a03eb10500d6e2f2d294)